plugins/PluginLoader.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+import os, inspect
+import importlib.util
+
+def load_plugins():
+    '''Loads all the plugins inside '/plugins' folder and then instantiate them,
+    returning it's objects.
+    '''
+    plugin_folder = "./plugins"
+    possible_plugins = os.listdir(plugin_folder)
+    plugins = []
+    plugins_instances = []
+
+    for plugin in possible_plugins:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)

Reply:
I'll change this in the next PR.